e2e/discussions.spec.ts

```diff
@@ -257,8 +236,12 @@ test.describe('Discussions', () => {
         // retry if the view count hasn't updated
         await expect(async () => {
           // reload is necessary for view count to update
-          await userPage.reload()
-          await expectDiscussionPageLoaded(userPage, userProject.id)
+          await reloadDashboardPage(userPage)
```

Comment:
Similar to the custom `goTo***` methods, use a custom `reload***` method that waits for SWC to fully load